automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Table.scala

```diff
@@ -110,4 +112,26 @@ case class Table(queryString: QueryString)(implicit webDriver: WebDriver)
     }
     map
   }
+
+  /**
+    * click button "Column Editor" invokes dropdown
+    *
+    * @return Css selector string that finds dropdown WebElement
+    */
+  private def clickColumnEditorButton(): CssSelectorQuery = {
+    // find dynamic dropdown id
+    val dropdownId = columnEditorButton.query.element.underlying.findElement(By.xpath("./..")).getAttribute("data-toggle")
+    val css = CssSelectorQuery(s"#${dropdownId}")
+    columnEditorButton.doClick()
+    Try(
+      await condition {
+        val dropdown: Option[Element] = find(css)
+        dropdown.get.underlying.getAttribute("class").contains("is-open")
+      }
+    ).recover {
+      case _ => columnEditorButton.doClick() // retry click when dropdown is not found or open
+    }
+    css
+  }
+
```

Comment:
wait for dropdown to appear after click. if none, retry click but don't wait for dropdown